tests/testdata/tolerations-dc.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: cassandra.datastax.com/v1beta1
+kind: CassandraDatacenter
+metadata:
+  name: dc1
+spec:
+  clusterName: cluster1
+  serverType: dse
+  serverVersion: "6.8.1"
+  managementApiAuth:
+    insecure: {}
+  size: 3
+  storageConfig:
+      cassandraDataVolumeClaimSpec:
+        storageClassName: server-storage
+        accessModes:
+          - ReadWriteOnce
+        resources:
+          requests:
+            storage: 1Gi
+  racks:
+    - name: r1
+  config:
+    jvm-server-options:
+      initial_heap_size: "800m"
+      max_heap_size: "800m"
+      additional-jvm-opts:
+        - "-Dcassandra.system_distributed_replication_dc_names=dc1"
```

Comment:
All three of these settings are necessary to fully bring up the Cassandra service in the recreated pod.